pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "offchain"
-version = "0.1.1"
+version = "0.0.0"
 description = "Open source metadata processing framework"
 authors = ["Zora eng <eng@zora.co>"]
 readme = "README.md"
 documentation = "https://ourzora.github.io/offchain"
 
 [tool.poetry.dependencies]
-python = ">=3.9"
+python = ">=3.9,<4"
```

Comment:
@mattmalec  this would be a break

Reply:
i think it's a break right now and this fixed it, im unable to `poetry install` since that version of web3 isn't compatible with python 4

Reply:
yep @mattmalec  I mean setting version to 0.0.0